src/main/java/uk/gov/hmcts/probate/service/notification/SmeeAndFordPersonalisationService.java

```diff
@@ -371,4 +371,11 @@ private String removeLastNewLine(String data) {
         return data.substring(0, data.lastIndexOf(NEW_LINE));
     }
 
+    private String getPoundValue(BigDecimal value) {
+        if (value == null || value.toString().length() < 2) {
+            return "0";
+        }
+        return value.toString().substring(0, value.toString().length() - 2);
+    }
+
```

Comment:
```suggestion
    private String getPoundValue(BigDecimal penceValue) {
        if (penceValue == null) {
            return "0";
        }
        final BigDecimal poundValue = penceValue.divideToIntegralValue(BigDecimal.valueOf(100L));
        return poundValue.toString();
    }
```

handles the (unexpected) case where we have fractional parts of a penny) and i think is slightly simpler to understand than taking a substring?